scripts/launch_dev.sh

```diff
@@ -1,4 +1,9 @@
-# can pass target port number to script for redis as: bash ./this_script.sh <new_port>
+# Script builds, connects, and runs the three components of this app: Server, Worker Pool, and Redis instance.
+
+# can pass override target port number to script for redis as: bash ./this_script.sh <new_port>
+
+# if you want more/fewer worker processes in the workerpool, please edit the 'numprocs' field in supervisord.conf
+
 # TODO redis doesn't currently like this and I'm not sure why
```

Comment:
this is slightly sus lol

Reply:
sus in a "too complicated for general users" way or sus in a "looks nefarious" sort of way?

Reply:
okay I'll make that issue